geppetto.js/geppetto-client/js/common/layout/utils.ts

```diff
@@ -13,9 +13,10 @@ export function widget2Node (configuration: Widget) {
     type: "tab",
     enableRename: false,
     enableClose,
+    panelName,
```

Comment:
why is this needed?

Reply:
It was being destructed but not used.
I assumed it was a mistake so I added it, but let me know if the mistake was me adding it :stuck_out_tongue:

Reply:
it doesn't hurt, let's keep it